js/qz-tray.js

```diff
@@ -615,7 +615,12 @@ var qz = (function() {
                     //disable secure ports if page is not secure
                     if (typeof location === 'undefined' || location.protocol !== 'https:') {
                         if (options == undefined) { options = {}; }
-                        options.usingSecure = false;
+
+                        //respect forcing secure ports if it is defined, otherwise disable
+                        if (options.usingSecure === 'undefined') {
```

Comment:
@dsanders11, typo... this should be either `=== undefined` (without single quotes) or `typeof ... === 'undefined'` otherwise the conditional fails.